node/actors/bft/src/replica/leader_prepare.rs

```diff
@@ -110,6 +111,14 @@ impl StateMachine {
             });
         }
 
+        // Replica MUSN'T vote for blocks which have been already pruned for storage.
+        // (because it won't be albe to persist and broadcast them once finalized).
```

Comment:
```suggestion
        // Replica MUSTN'T vote for blocks which have been already pruned for storage.
        // (because it won't be able to persist and broadcast them once finalized).
```